modules/ROOT/partials/configuration/force_hex_color.adoc

```diff
@@ -0,0 +1,21 @@
+[[force_hex_color]]
+== `+force_hex_color+`
+
+This option allows the integrator to optionally force the conversion of either RGB or HEX colors.
+
+*Type:* `+String+`
+
+*Default value:* `+'off'+`
+
+*Possible values:* `+'always'+`, `+'rgb_only'+`, `+'off'+`
```

Comment:
I think we need to describe what these three settings does or at lest the first two. So `always` will convert all colors RGB and RGBA to hex colors. `rgb_only` will only convert RGB values but keep RGBA values as they are.